LibThreatClassic2.lua

```diff
@@ -1175,6 +1320,67 @@ do
 			self:SendCommRaw(self:GroupDistribution(), "THREAT_UPDATE", petMsg)
 		end
 		lastPublishTime = GetTime()
+
+		self:PublishMeleeRange(force)
```

Comment:
please remove

Reply:
meele range is only relevant for aggroMod calculations, which in turn are only relevant for threatPercentage calculations. It doesn't have to be included at so many places.